db/seeds.rb

```diff
@@ -5,8 +5,8 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
-julie = Candidate.create!(name: "Julie Andrews", description: "Wonderful")
-olo = Candidate.create!(name: "Olo Wigley", description: "Baby")
+julie = Candidate.create!(name: "Julie Andrews", photo_url: "http://www.heartandsoul.com/wp-content/uploads/2013/11/TamaraNatalieMaddenHeadshot.jpg", twitter: "HottestStartups", website: "http://t.co/3gDtETAFpu")
```

Comment:
Just FYI, Hound is expecting something like:

```ruby
julie = Candidate.create!(
  name: "Julie Andrews",
  photo_url: "http://www.heartandsoul.com/wp-content/uploads/2013/11/TamaraNatalieMaddenHeadshot.jpg",
  twitter: "HottestStartups", website: "http://t.co/3gDtETAFpu",
)
```

The photo url line will still be too long, but I don't think it's worth breaking it up any further.